EIPS/eip-2937.md

```diff
@@ -0,0 +1,35 @@
+---
+eip: 2937
+title: SET_INDESTRUCTIBLE opcode
+author: Vitalik Buterin (@vbuterin)
+discussions-to: https://ethereum-magicians.org/t/eip-2937-set-indestructible/4571
+status: Draft
+type: Standards Track
+category: Core
+created: 2020-09-04
+---
+
+## Simple Summary
+
+Add a `SET_INDESTRUCTIBLE (0xA8)` opcode that prevents the contract from calling `SELFDESTRUCT (0xff)`.
+
+## Specification
```

Comment:
```suggestion
## Abstract

## Specification
```
Try to match the template whenever possible as it allows us to do fancy things with automated tooling in the future.